src/ngrx/12-test-reducer/test-reducer.md

```diff
@@ -30,9 +30,57 @@ The `--watch` switch will rerun your tests whenever a code file changes. You can
 
 ## Description
 
-TODO: remove or use parts of for an intro for the section
+When testing a Reducer, we will test each of its `on()` handlers by verifying each pure function returns the expected value given an Action and a Login State (typically the `initialState`). Unlike the previous testing sections, when testing the Reducer, we don't need a `TestBed` since there shouldn't be any dependencies when dealing with the Reducer.
```

Comment:
typically or by default?

Reply:
There's no default since the schematic doesn't give you any examples using initialState. You don't have to use initialState (for example resetting the state when user logs out)